test/unit/mongo/MongoTripodViewsTest.php

```diff
@@ -521,6 +522,116 @@ public function testGenerateViewWithTTL()
         $this->assertInstanceOf('\MongoDB\BSON\UTCDateTime', $actualView['_cts']);
     }
 
+    public function testNonExpiringViewWithNegativeTTL()
+    {
+        $views = new \Tripod\Mongo\Composites\Views(
+            $this->viewsConstParams[0],
+            $this->viewsConstParams[1],
+            $this->viewsConstParams[2]
+        );
+
+        $view = $views->getViewForResource(
```

Comment:
This isn't used - should it be tested?

Reply:
It's used indirectly, that's how we ensure that the view has actually been generated.